docs/sink_config_options.rst

```diff
@@ -3,6 +3,19 @@
 JDBC Sink Configuration Options
 -------------------------------
 
+Security
```

Comment:
nit: maybe emphasize which connection this security refers to? Kafka security is via worker config, but new users may not always realize that. maybe "Database Connection Security" or something like that?